swarm/CHANGELOG.md

```diff
@@ -14,11 +14,16 @@
   This variant was never constructed and thus dead code.
   See [PR 3605].
 
+- Deprecate `ConnectionHandlerUpgrErr` in favor of `StreamUpgradeError`.
+  This new type features a flat interface which should be easier to use.
+  See [PR 3882].
```

Comment:
Changelog reads like the `ConnectionHandlerUpgrErr` doesn't change. Though given that it is a type alias, it is flattened just like `StreamUpgradeError`.

Would something along the lines below help?
```suggestion
- Flatten `ConnectionHandlerUpgrErr` and rename to `StreamUpgradeError`.
  See [PR 3882].
```

Reply:
Sounds good to me!